### DISCUSSION TITLE
Archiving Top Level Roadmap and pointing to GitHub Projects

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-core#9866] [General]
I would like to archive the [Top-Level-Roadmap](https://github.com/micronaut-projects/micronaut-core/wiki/Top-Level-Roadmap) and point users to the [GitHub Projects](https://github.com/orgs/micronaut-projects/projects). 

**Why?** We don't keep the Top-Level-Roadmap wiki page up-to-date and I think pointing them to the GitHub Projects reflects better the current development roadmap. 

@micronaut-projects/core-developers  Thoughts?

**Comment by wetted** [comment]
This makes sense sense to me.

Maybe a longer term roadmap could be published as part of the information at https://micronaut.io/foundation/, since it seems to me the advisory board would drive that.

**Reply by sdelamo** [reply]
I am not a fan of promising long-term roadmaps. They create an illusion of a promise that you will break. 

Having a roadmap for the following minors and the next major version is enough. I think that we can capture that via GitHub projects.

**Reply by wetted** [reply]
Fair enough. I mentioned it because I think it was done in the past.
